release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2582,3 +2582,107 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
+`templateinstance` can now create projects as defined in templates. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1713982[*BZ#1713982*])
+
+* Redeployment of certificates did not recreate the `ansible-service-broker` pod
+secrets, causing the service catalog to fail. A new playbook has been created to
+support updating the certificates. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1715322[*BZ#1715322*])
+
+* The IPv4 dictionary was recently modified and MTU was set incorrectly as a
+result. This IPv4 conditional has been removed, and now MTU is established
+correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1719362[*BZ#1719362*])
+
+* The `pom.xml` of some of the OpenShift Jenkins plugins had `http://` references
+instead of `https://` references for some of it's build time dependencies, and
+dependency downloads would occur over `http` instead of the `https` protocol.
+The `pom.xml` references have now been corrected and dependency downloads only
+occur using the `https` protocol. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1719477[*BZ#1719477*])
+
+* The readiness probe for `ElasticSearch` `curl` commands used `NSS`, which bloated
+the dentry cache. This would cause `ElasticSearch` to become unresponsive. To
+resolve this, set the `NSS_SDB_USE_CACHE=no` flag in the readiness probe to work
+around the dentry cache bloating. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1720479[*BZ#1720479*])
+
+* Previously, the web console showed a misleading warning that metrics might not
+configured for horizontal pod autoscalers when only the metrics server had been
+set up. The warning has been removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1721428[*BZ#1721428*])
+
+* Previously, the `image-signature-import` controller would only import up to
+three signatures, but the registry would often have more than three signatures.
+This would cause importing signatures to fail. The limit of signatures has been
+increased, and importing signatures from `registry.redhat.io` complete
+successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1722581[*BZ#1722581*])
+
+* The `prerequisites` playbook would fail because default values were not loaded
+correctly, causing sanity checks to fail. A step to run `openshift_facts` has
+been added to load all the default values, and sanity checks complete
+successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1724718[*BZ#1724718*])
+
+* `Kibana` would present a blank page or timeout if a large number of projects
+were creating too many calls to the `ElasticSearch` cluster, resulting in the
+timeout before a response is returned. Now, API calls are cached and processing
+is more efficient, reducing the opportunity for page timeouts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1726433[*BZ#1726433*])
+
+[[ocp-3-11-129-enhancements]]
+==== Enhancements
+
+* Service catalogs did not have a `redeploy-certificate` playbook. The
```

Comment:
I don't think it's appropriate to say service catalogs plural. Should be just "The service catalog did not have..."

Reply:
Fixed!